### PR TITLE
Migrate OpenAI & Anthropic plugins

### PR DESCRIPTION
**Pull request by Dabolus**
This PR migrates the more up-to-date OpenAI & Anthropic plugins from my repo so that from now on we can maintain them in this repo instead.

**Note: due to the regeneration of the lockfile, the PR also bumps the Cohere AI package to version `7.10.0`, which includes some breaking changes to the types. For this reason, a fix on the types of the Cohere plugin is also included in this PR.**

### CODE REVIEW

**Comment by EPMatt** [comment]
@all-contributors please add @Dabolus as a code contributor

**Review by EPMatt — CHANGES_REQUESTED** [review]
Thanks for the merging. Added a couple of edits to perform before merging

File: plugins/openai/package.json [hunk 1]
```diff
@@ -14,28 +14,30 @@
...
     "generative-ai"
   ],
-  "version": "0.3.0",
+  "version": "0.7.1",
   "type": "commonjs",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/TheFireCo/genkit-plugins.git",
-    "directory": "plugins/openai"
+    "directory": "plugins/anthropic"
```
[the remark this answers is not in the document]
Comment (Dabolus) [reply plugins/openai/package.json]: Nice catch, it was a copy paste 👀

**Review by EPMatt — APPROVED** [review]

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+As the plugins are currently in a very early stage, only their latest version will
+receive security patches, so please make sure to update the plugins frequently.
+Please note that breaking changes on the plugins themselves are currently not expected,
+but they might be required in case of breaking changes in Genkit or in the respective
+official SDKs. In either case, a migration guide will be provided to ease the transition
+to the latest version.
+
+## Reporting a Vulnerability
+
+If you find a vulnerability, please [open a bug report](https://github.com/TheFireCo/genkit-plugins/issues/new?labels=bug&template=bug_report.md)
```
[the remark this answers is not in the document]
Comment (Dabolus) [reply SECURITY.md]: Done 👍🏻

**Review by EPMatt — APPROVED** [review]
seems fine for the monorepo setup! @davidoort it would be great if you could review the changes to the ts source files in both plugins as you are more experienced with the plugins

**Review by davidoort — COMMENTED** [review]
Amazing work @Dabolus !!!

Since you've been added to CODEOWNERS for Anthropic and OpenAI I guess you're not adding a note in the respective plugin readme's? We can always add it later and do it for all of them ofc

**Review by davidoort — APPROVED** [review]
Ready to merge on my side, if tests pass and basic E2E examples have been tested

**Comment by EPMatt** [comment]
Thanks for the great contribution @Dabolus! @all-contributors please add him as a docs and test contributor as well.

**Pull request merged**